Sources/Controllers/2.0/RoutePlanning/OASelectTrackFolderViewController.m

```diff
@@ -229,7 +224,7 @@ - (CGFloat)tableView:(UITableView *)tableView estimatedHeightForRowAtIndexPath:(
 {
         NSDictionary *item = _data[indexPath.section][indexPath.row];
         NSString *cellType = item[@"type"];
-        if ([cellType isEqualToString:kMultiIconTextDescCell])
+        if ([cellType isEqualToString:[OAMultiIconTextDescCell getCellIdentifier]])
```

Comment:
Strange code for ```estimatedHeightForRowAtIndexPath```. Estimated row height should not be UITableViewAutomaticDimension at all.